.circleci/cimodel/caffe2_build_definitions.py

```diff
@@ -12,27 +12,22 @@
 DOCKER_IMAGE_VERSION = 248
 
 
-# TODO Sort the config.yml upstream so the ubuntu configs are contiguous
 CONFIG_HIERARCHY = [
+    (Ver("ubuntu", "14.04"), [
```

Comment:
We use ubuntu and a version here and xenial/trusty below. We could probably string replace one into the other pretty quickly. Or maybe we can create python classes for os types, compilers, cuda versions, backends or other things. Not sure if that would actually buy us much. Might be simpler to just rename xenial/trustys to ubuntus for consistency.